src/types/index.ts

```diff
@@ -396,6 +396,8 @@ export interface ITransaction {
   pendingTokenTx?: boolean;
   rawTx?: any; // TODO find type
   tipUrl?: string;
+  sent?: boolean;
```

Comment:
How the account can be `sent`? I guess this is a mistake.

Reply:
i added TODO for this line